collector/hadoop/Makefile

```diff
@@ -0,0 +1,12 @@
+DOCKER_NETWORK = hadoop_integrations
+ENV_FILE = hadoop.env
+tag := 2.0.0-hadoop3.2.1-java8
+
+wordcount:
+	docker build -t hadoop-wordcount ./submit
+	docker run --network ${DOCKER_NETWORK} --env-file ${ENV_FILE} bde2020/hadoop-namenode:$(tag) hdfs dfs -mkdir -p /input/
```

Comment:
Could you add some explanatory comments regarding what each of these lines should accomplish?

Reply:
@nslaughter Addressed your comments.